embedded-bins/iptables/Dockerfile

```diff
@@ -1,12 +1,14 @@
 ARG BUILDIMAGE
 FROM $BUILDIMAGE AS build
 
-RUN apk add build-base git file curl \
-	linux-headers pkgconf bison flex libmnl-dev libmnl-static libnftnl-dev
+RUN apk add build-base curl pkgconf \
+	linux-headers \
+	libmnl-dev libmnl-static \
+	libnftnl-dev
```

Comment:
Not sure if we should build those dependencies from scratch. We're not controlling the versions in any way here, and are just taking the stuff that's in the Alpine package repos at build time. This has been tackled a bit in #2124 (5391cddb8d80c8e1d3a8328ce56daf63db4ae287) for technical reasons, but has been [discarded](https://github.com/k0sproject/k0s/pull/2124#discussion_r960630241).

We're also building libseccomp from scratch when building runc, even if [libseccomp.a is available in the Alpine repos](https://pkgs.alpinelinux.org/contents?file=libseccomp.a&path=&name=&branch=v3.17).

Reply:
I don't think we need build all deps from scratch unless there is a good reason for it.

libseccomp has been problematic in the past so I think it makes sense to build that.